graders/c/cgrader/cgrader.py

```diff
@@ -74,12 +74,19 @@ def __init__(self):
 
 
 class CGrader:
-    def __init__(self, compiler="gcc"):
+    def __init__(self, compiler: str = "gcc") -> None:
         with open(DATAFILE) as file:
             self.data = json.load(file)
         self.compiler = compiler
 
-    def run_command(self, command, input=None, sandboxed=True, timeout=None, env=None):
+    def run_command(
+        self,
+        command: str | list[str],
+        input: Any | None = None,
+        sandboxed: bool = True,
+        timeout: float | None = None,
+        env: subprocess._ENV | None = None,
+    ):
```

Comment:
Is the return type here just a string? A few of these methods don't have return types annotated.

Reply:
Resolved in faa4b8f.